recipes/_nfs_config.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: _nfs_config
```

Comment:
let's drop the leading underscore. We are going to remove it from all recipes soon